### PR TITLE
Add goimports to linters

### PR DESCRIPTION
**Pull request by sp-roger-clotet**

### CODE REVIEW

File: makefile_sample/Makefile [hunk 1]
```diff
@@ -35,6 +35,14 @@ fmt-ci:
 	fgt gofmt -l $(SOURCES)
 .PHONY: fmt-ci
 
+imports:
+	goimports -w $(SOURCES)
+.PHONY: imports
+
+imports-ci:
+	fgt goimports -w $(SOURCES)
```
Comment (gonzaloserrano) [inline makefile_sample/Makefile]: i think in `-ci` mode should be `-l` instead of `-w`, otherwise it's never reported as an error. What do you think?

Comment (rogerclotet) [inline makefile_sample/Makefile]: You're right, I made a mistake copying from my repo. Fixing it.

Comment (gonzaloserrano) [inline makefile_sample/Makefile]: Also, could you update the README linters part? https://github.com/gonzaloserrano/go-nzalo/blob/master/makefile_sample/README.md

Comment (rogerclotet) [inline makefile_sample/Makefile]: Fixed.

**Comment by gonzaloserrano** [comment]
Thanks for the contrib!

**Pull request merged**